integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/environment/EnvironmentTestDto.java

```diff
@@ -185,8 +186,11 @@ public EnvironmentTestDto withS3Guard(String tableName) {
     }
 
     public EnvironmentTestDto withS3Guard() {
-        String tableName = dynamoTableName + '-' + UUID.randomUUID().toString();
-        return withS3Guard(tableName);
+        if (CloudPlatform.AWS.equals(getTestContext().getCloudProvider().getCloudPlatform())) {
+            String tableName = dynamoTableName + '-' + UUID.randomUUID().toString();
+            return withS3Guard(tableName);
+        }
+        return this;
```

Comment:
would be a good idea to drop a warn message here into the logs, don't you think?

Reply:
In personal communication we agreed to place a log line with severity INFO. Thanks!